scripts/Install

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+
+# Program information
+prog_name="BorgBackup"
+version="1.1.10"
+signature="BSD License (BSD)"
+git_url="https://github.com/borgbackup/borg"
+installation_path=/usr/local/bin
+
+# System information
+SYS=$(uname -s | awk '{print tolower($0)}')
+OS=$(awk '/DISTRIB_ID=/' /etc/*-release | sed 's/DISTRIB_ID=//' | tr '[:upper:]' '[:lower:]')
+ARCH=$(uname -m | sed 's/x86_//;s/i[3-6]86/32/')
+VERSION=$(awk '/DISTRIB_RELEASE=/' /etc/*-release | sed 's/DISTRIB_RELEASE=//' | sed 's/[.]0/./')
+
+
+
+install_all_dependencies(){
+  if [[ $OS == 'debian' || $OS == 'ubuntu' ]]; then
+    sudo apt-get install -y python3 python3-dev python3-pip python-virtualenv \
+    libssl-dev openssl \
+    libacl1-dev libacl1 \
+    build-essential
+    sudo apt-get install -y libfuse-dev fuse pkg-config
+
+  elif [[ $OS == 'arch' ]]; then
+    sudo pacman -S python3 python3-dev python3-pip python-virtualenv \
+    libssl-dev openssl \
+    libacl1-dev libacl1 \
+    build-essential libfuse-dev \
+    fuse pkg-config
+
+  elif [[ $OS == 'fedora' ]]; then
+    sudo dnf install python3 python3-devel python3-pip python3-virtualenv
+    sudo dnf install openssl-devel openssl
+    sudo dnf install libacl-devel libacl
+    sudo dnf install gcc gcc-c++
+    sudo dnf install redhat-rpm-config
+    sudo dnf install fuse-devel fuse pkgconfig
+
+  elif [[ $OSTYPE == 'darwin' ]]; then
+    brew install python3 openssl
+    brew install pkg-config
+    pip3 install virtualenv
+    brew cask install osxfuse
+
+  elif [[ "$OSTYPE" == "freebsd"* ]]; then
+    pkg install -y python3 openssl fusefs-libs pkgconf
+    pkg install -y git
+    python3.4 -m ensurepip # to install pip for Python3
+    To use the mount command:
+    echo 'fuse_load="YES"' >> /boot/loader.conf
+    echo 'vfs.usermount=1' >> /etc/sysctl.conf
+    kldload fuse
+    sysctl vfs.usermount=1
+
+  else; then
+    echo \n "We can't install the dependencies on this system"
+    echo "Try manually" \n
+
+  fi
+}
+
+install_dependencies(){
```

Comment:
this is the stuff you use to download/verify the borg binary, name your function appropriately.